cli/src/main.rs

```diff
@@ -234,6 +229,57 @@ pub(crate) struct CoreOptions {
     core: usize,
 }
 
+#[derive(clap::Parser)]
+pub(crate) struct FormatOptions {
+    #[clap(value_enum, ignore_case = true, default_value = "elf", long)]
+    format: Format,
+    /// The address in memory where the binary will be put at.
+    #[clap(long)]
+    pub base_address: Option<u64>,
+    /// The number of bytes to skip at the start of the binary file.
+    #[clap(long, default_value = "0")]
+    pub skip: u32,
+    /// The idf bootloader path
+    #[clap(long)]
+    pub bootloader: Option<PathBuf>,
```

Comment:
Maybe we should name this option something like `idf-bootloader`, so it's clear that it's related to IDF?

To avoid confusion with potential other bootloaders.